### DISCUSSION TITLE
Minimize Options

### DISCUSSION THREAD

**Discussion by Hotair10** [Chatterino/chatterino2#4179] [Ideas] [answered]
In previous versions of Chatterino, when minimizing a popup chat, it would simply disappear into the taskbar. This paired perfectly with my "Always on Top" setting I had enabled, as it would be out of the way when I didn't want it blocking my screen, such as playing a game on that monitor.
In the newest update, it now minimizes into a little tab above my taskbar like this:
![image](https://user-images.githubusercontent.com/5515786/203149145-bf69c5a9-5420-4bc9-b8e7-afaccea534c4.png)

With the "Always on Top" setting enabled, this little tab is also always on top. While the tab can be moved around freely, it'd be nice if there was an option to disable the feature entirely and have it operate as it did in previous versions.

As a side note, it continues to add additional blips for each popup chat you minimize. No information on channel or anything, just a repeated list of the application title, making the feature even less useful than it could be.
![image](https://user-images.githubusercontent.com/5515786/203149669-6d7344b2-e14a-44f2-9746-b139f5f24b06.png)

tl;dr: Add a setting for old minimize behavior to still be an option.

**Comment by Hotair10** [comment]
Just for completeness and clarity, this is how the taskbar worked on previous versions. Minimizing removed it from the screen and you'd pull it back up from here.
![image](https://user-images.githubusercontent.com/5515786/203151817-234f7485-9995-408d-9285-5a8a40b03486.png)

**Comment by Felanbird** [comment]
#3863

**Reply by Hotair10** [reply]
This isn't the same issue, though. This has to do with the new minimize behavior creating the little tabbed window when it did not before.

**Reply by Felanbird** [reply]
Interesting, it's not the same issue but it's likely they are both caused by the same change

**Comment by Felanbird** [comment]
This is unintended bug so I've moved it to an issue #4180

**Comment by Felanbird** [comment, marked as answer]
@Hotair10 Your issue + the other issue I mentioned earlier, have been been resolved in https://github.com/Chatterino/chatterino2/pull/4181, as you are a Chatterino stable user, you'll get the fix next week when 2.4.0 officially comes out.

If you'd like the fix immediately, you can use the [nightly build](https://github.com/Chatterino/chatterino2/releases/tag/nightly-build) once this comment is 30 minutes old.

More info on the nightly build can be found [on our wiki](https://wiki.chatterino.com/Help/#what-is-nightly-and-how-to-use-install-it) and full install instructions in case you run into any issues can be found [here](https://felanbird.github.io/Chatterino-Nightly-Install-Guide/)

**Reply by Hotair10** [reply]
I actually use the beta build, but have switched to the stable build until this is fixed. I'll stay away from nightly for now as I rely on Chatterino and like to avoid most hiccups if I can. Thanks for working so quickly on it and I look forward to seeing it pushed out!
